layers.go

```diff
@@ -136,9 +136,12 @@ type Layer struct {
 	TOCDigest digest.Digest `json:"toc-digest,omitempty"`
 
 	// UncompressedSize is the length of the blob that was last passed to
-	// ApplyDiff() or create(), after we decompressed it.  If
-	// UncompressedDigest is not set, this should be treated as if it were
-	// an uninitialized value.
+	// ApplyDiff() or create(), after we decompressed it.
+	//
+	// - If UncompressedDigest is set, this must be set to a valid value.
+	// - Otherwise, if TOCDigest is set, this is either valid or -1.
+	// - If neither of this digests is set, this should be treated as if it were
+	//   an uninitialized value.
```

Comment:
You need indentation for list items (same as for code blocks, i.e. an extra space).

```suggestion
	//  - If UncompressedDigest is set, this must be set to a valid value.
	//  - Otherwise, if TOCDigest is set, this is either valid or -1.
	//  - If neither of this digests is set, this should be treated as if it were
	//    an uninitialized value.
```

Reply:
Yeah, `gofmt` didn’t want to help with this (and struct field comments are formatted in HTML just as an uninterpreted code block), so I punted.

You’re right, this is the right thing to do. Fixed in https://github.com/containers/storage/pull/2136 .